service/src/main/java/bio/terra/catalog/iam/SamService.java

```diff
@@ -50,7 +50,7 @@ public boolean hasGlobalAction(AuthenticatedUserRequest userRequest, SamAction a
     ResourcesApi resourceApi = resourcesApi(accessToken);
     try {
       return SamRetry.retry(
-              () -> resourceApi.resourceActions(CATALOG_RESOURCE_TYPE, CATALOG_RESOURCE_ID))
```

Comment:
It looks like SonarQube is complaining about a stray `CATALOG_RESOURCE_ID` somewhere in the code. Probably worth removing it if it isn't relevant.